tests/plugins/elementspath/elementspath.js

```diff
@@ -28,6 +28,19 @@
 		'test widget mockup element path': function() {
 			this.editorBot.setHtmlWithSelection( bender.tools.getValueAsHtml( 'testWidgetSample' ) );
 			this.assertPath( 'div,figcaption' );
+		},
+
+		// (1191)
+		'test elements path items not draggable': function() {
+			this.editorBot.setHtmlWithSelection( '<ul><li>^</li></ul>' );
+
+			var path = this.editor.ui.space( 'path' ),
+				elements = path.getElementsByTag( 'a' ).toArray();
+
+			CKEDITOR.tools.array.forEach( elements, function( element ) {
+				assert.areEqual( 'false', element.getAttribute( 'draggable' ), 'Element draggable attribute value should be "false".' );
```

Comment:
It is best practice to not repeat the expected value in assertion message. That way you reduce the risk of someone missing this when changing expected in the future.